src/Microsoft.IdentityModel.JsonWebTokens/JsonWebTokenHandler.cs

```diff
@@ -999,32 +999,29 @@ public virtual TokenValidationResult ValidateToken(string token, TokenValidation
         /// <returns></returns>
         private TokenValidationResult ValidateToken(string token, JsonWebToken outerToken, string decryptedJwt, TokenValidationParameters validationParameters)
         {
-            var validationParametersCopy = validationParameters;
-            if (validationParametersCopy.ConfigurationManager != null)
+            BaseConfiguration currentConfiguration = null;
+            if (validationParameters.ConfigurationManager != null)
             {
                 try
                 {
-                    // TODO: Use overloads to get rid of the Clone() operation.
-                    var configuration = validationParametersCopy.ConfigurationManager.GetBaseConfigurationAsync(CancellationToken.None).ConfigureAwait(false).GetAwaiter().GetResult();
-                    validationParametersCopy = validationParameters.Clone();
-                    validationParametersCopy.Configuration = configuration;
+                    currentConfiguration = validationParameters.ConfigurationManager.GetBaseConfigurationAsync(CancellationToken.None).ConfigureAwait(false).GetAwaiter().GetResult();
```

Comment:
Referenced this PR in this issue: https://github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/issues/468